utils/gpg.py

```diff
@@ -0,0 +1,18 @@
+import os
+import base64
+
+from subprocess import PIPE, Popen, STDOUT
+
+
+def gpg_key_valid(public_gpg_key):
+    DENNULL = open(os.devnull, 'w')
```

Comment:
not a big deal, but `DENNULL` vs `DEVNULL`

Reply:
Thanks!